src/Provider/AbstractProvider.php

```diff
@@ -537,7 +537,9 @@ protected function parseResponse(ResponseInterface $response)
         $content = (string) $response->getBody();
         $type = $this->getContentType($response);
 
-        if (strpos($type, "json") !== false) {
+        // JSON  = application/json
+        // JSONP = application/javascript
+        if (strpos($type, "json") !== false || strpos($type, "javascript") !== false) {
```

Comment:
Since you're working on this, can you change these double quotes to be single? Just a pet peeve of mine that I didn't notice when it was changed the first time around.